tools/postgres/schema.sql

```diff
@@ -32,7 +32,7 @@ CREATE TABLE webknossos.analytics(
 
 CREATE TYPE webknossos.ANNOTATION_TRACING_TYPE AS ENUM ('skeleton', 'volume');
 CREATE TYPE webknossos.ANNOTATION_TYPE AS ENUM ('Task', 'Explorational', 'TracingBase', 'Orphan');
-CREATE TYPE webknossos.ANNOTATION_STATE AS ENUM ('Active', 'Finished', 'Cancelled');
```

Comment:
Can we keep the initial schema also in the `evolutions` folder? This helps to keep track how the schema evolved.

Reply:
yes, good idea